packages/components/tabs/src/TabsContext.tsx

```diff
@@ -0,0 +1,22 @@
+import type { TabsProps } from '@radix-ui/react-tabs'
```

Comment:
The context for the radio was organized in a different way https://github.com/adevinta/spark/blob/main/packages/components/radio/src/RadioGroupProvider.tsx. Should we come with agreement regarding this ?

Reply:
You're right. We may be interested in future refactoring, with a context creator util, that would help us achieve that... wdyt?

Reply:
👍🏻